src/lib.rs

```diff
@@ -259,6 +259,23 @@ impl<T: Runtime> Client<T> {
         &self.genesis_hash
     }
 
+    /// Return rpc.
+    pub fn rpc(&self) -> &Rpc<T> {
+        &self.rpc
+    }
+
+    /// Request method wrapper.
+    pub async fn request<Ret>(
+        &self,
+        method: impl Into<String>,
+        params: impl Into<jsonrpsee::common::Params>,
+    ) -> Result<Ret, jsonrpsee::client::RequestError>
+    where
+        Ret: jsonrpsee::common::DeserializeOwned,
+    {
+        self.rpc().client().request(method, params).await
+    }
```

Comment:
This seems more like a convenience method and could perhaps live in your own code as well? Or why do you think this is important to add?
My concern here is that this leaks types from `jsonrpsee`, which makes it harder for us to change the implementation under the hood (although we likely already leak plenty in other places but the less we do it the better).

Reply:
Many rpc methods are not covered by subxt, and there is no obvious way except changing library code to add customized call. For example, I have to change Client implementation if I want to use `chain_getHead`. Further more, substrate allows user to add their own runtime module which may contain extra private rpc methods. 

In conclusion, It needs to expose an api for user to make their own private rpc call.